Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/function/CreateFunctionAction.java

```diff
@@ -147,7 +147,7 @@ public void actionPerformed(ListingActionContext context) {
 			}
 		}
 		else {
-			cmd = new CreateFunctionCmd(name, entry, body, SourceType.USER_DEFINED, allowExisting,
+			cmd = new CreateFunctionCmd(null, entry, body, SourceType.USER_DEFINED, allowExisting,
```

Comment:
You can also remove the declaration of name on line 120

Reply:
Correct, thanks for noticing